### PR TITLE
fix: Handle panic on Waf logging configuration not found

### PR DESCRIPTION
**Pull request by zagronitay**
🎉 Thank you for making CloudQuery awesome by submitting a PR 🎉

#### Summary

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [ ] Update or add tests. Learn more about testing [here](https://docs.cloudquery.io/docs/developers/sdk/testing) 🧪
- [ ] Update the docs by running `go run ./docs/docs.go` and committing the changes 📃
- [ ] If adding a new resource, add [relevant Terraform files](../terraform) in a separate PR 📂
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

File: resources/services/waf/web_acls.go [hunk 1]
```diff
@@ -250,6 +255,9 @@ func fetchWafWebACLLoggingConfiguration(ctx context.Context, meta schema.ClientM
 }
 func resolveWafWebACLLoggingConfigurationRedactedFields(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
 	conf := resource.Item.(*types.LoggingConfiguration)
+	if conf == nil {
```
Comment (yevgenypats) [inline resources/services/waf/web_acls.go:258]: I don't think this is needed.

File: resources/services/waf/web_acls.go [hunk 2]
```diff
@@ -187,14 +187,19 @@ func fetchWafWebAcls(ctx context.Context, meta schema.ClientMeta, _ *schema.Reso
 				var exc *types.WAFNonexistentItemException
 				if errors.As(err, &exc) {
 					if exc.ErrorCode() != "WAFNonexistentItemException" {
-						return err
+						return diag.WrapError(err)
 					}
 				}
```
Comment (yevgenypats) [inline resources/services/waf/web_acls.go:192]: unless I missed some edge-case I think it should be a `continue` here

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Either I missed an edge or I think there is a better option to fix it with oneline instead of adding not nil everywhere

File: resources/services/waf/web_acls.go (hunk 1 above)
Comment (bbernays) [reply resources/services/waf/web_acls.go:258]: Does marshalling a nil value give you valid JSON?

**Review by roneli — APPROVED** [review]
LGTM

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**